packages/editor/src/store/actions.js

```diff
@@ -59,7 +59,11 @@ export function* setupEditor( post, edits, template ) {
 		content = post.content.raw;
 	}
 
-	let blocks = parse( content );
+	let blocks = parse( content, {
```

Comment:
Is there any plan to allow, plugin developers to create custom sources and use them in their blocks?